diagnostic.js

```diff
@@ -15,7 +15,16 @@ db.dropDatabase();
 // House Targaryen, motto 'Fire and Blood'
 
 // your code begins here
-
+db.createCollection(houses {[
+  {'name': 'House Arryn',
+  'motto':'As high as honor'}
+]
+  [{'name': 'House Stark',
+  'motto':'Winter is coming'}
+]
+  [{'name': 'House Targaryen',
+  'motto':'Fire and blood'}
+]})
```

Comment:
Your syntax is not quite right. Take a look at the documentation again: https://docs.mongodb.com/v3.2/reference/method/db.createCollection/

Note that you need a comma separating the name of the collection, and the objects that are going inside. Take a look at this example. Note that there are NO `[ ]`s. 

```js
db.createCollection(houses, { 
   {name: 'House Arryn', 'motto': 'As high as honor'}
}
```